Buttplug/Server/Bluetooth/Devices/Cueme.cs

```diff
@@ -0,0 +1,213 @@
+﻿// <copyright file="Cueme.cs" company="Nonpolynomial Labs LLC">
+// Buttplug C# Source Code File - Visit https://buttplug.io for more info about the project.
+// Copyright (c) Nonpolynomial Labs LLC. All rights reserved.
+// Licensed under the BSD 3-Clause license. See LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Timers;
+using Buttplug.Core;
+using Buttplug.Core.Devices;
+using Buttplug.Core.Logging;
+using Buttplug.Core.Messages;
+
+namespace Buttplug.Server.Bluetooth.Devices
+{
+    internal class CuemeBluetoothInfo : IBluetoothDeviceInfo
+    {
+        public enum Chrs : uint
+        {
+            Tx = 0,
+        }
+
+        public Guid[] Services { get; } = { new Guid("0000fff0-0000-1000-8000-00805f9b34fb") };
+
+        public string[] NamePrefixes { get; } = { "FUNCODE_" };
+
+        public string[] Names { get; } = { };
+
+        public Dictionary<uint, Guid> Characteristics { get; } = new Dictionary<uint, Guid>()
+        {
+            { (uint)Chrs.Tx, new Guid("0000fff1-0000-1000-8000-00805f9b34fb") },
+        };
+
+        public IButtplugDevice CreateDevice(IButtplugLogManager aLogManager,
+            IBluetoothDeviceInterface aInterface)
+        {
+            return new Cueme(aLogManager, aInterface, this);
+        }
+    }
+
+    internal class Cueme : ButtplugBluetoothDevice
+    {
+        internal struct CuemeType
+        {
+            public string Name;
+            public uint VibeCount;
+        }
+
+        private static readonly Dictionary<uint, CuemeType> DevInfos = new Dictionary<uint, CuemeType>
+        {
+            { 1, new CuemeType() { Name = "Mens", VibeCount = 8 } },
```

Comment:
Can you add some sort of documentation about order of vibrators here? Just saying "there are 8 vibrators" doesn't really tell us much. Even stpihkal links would be fine.